modules/calc/src/main/java/com/opengamma/strata/calc/Column.java

```diff
@@ -27,100 +26,88 @@
 import com.opengamma.strata.calc.config.MarketDataRules;
 import com.opengamma.strata.calc.config.Measure;
 import com.opengamma.strata.calc.config.ReportingCurrency;
-import com.opengamma.strata.calc.config.pricing.PricingRules;
+import com.opengamma.strata.calc.runner.CalculationParameter;
+import com.opengamma.strata.calc.runner.CalculationParameters;
 import com.opengamma.strata.collect.ArgChecker;
 
 /**
- * Defines a column in a set of calculation results. A column specifies a measure and may specify
- * overrides for the pricing rules and market data rules.
+ * Defines a column in a set of calculation results.
+ * <p>
+ * {@link CalculationRunner} provides the ability to calculate a grid of results
+ * for a given set targets and columns. This class is used to define the columns.
+ * <p>
+ * A column is defined in terms of a unique name, measure to be calculated and
+ * a set of parameters that control the calculation. The functions to invoke
+ * and the default set of parameters are defined on {@link CalculationRules}.
  */
 @BeanDefinition
 public final class Column implements ImmutableBean {
```

Comment:
The design is essentially to use parameters to tweak the behaviour of standard functions, rather than to allow the function to be changed. It may involve more function writing, or it may not, we don't really know yet. What we can say is that function overrides could be added later.